src/generator/app.js

```diff
@@ -1,18 +1,32 @@
 'use strict';
 
 var express = require('express');
+var multer = require('multer');
 var app = express();
 
+var upload = multer({ dest: 'uploads/' });
+
 var generator = require('./backend/generator.js');
 
+var uploadedFiles = upload.fields([
+  {name: 'speakerfile', maxCount: 1},
+  {name: 'sessionfile', maxCount: 1},
+  {name: 'trackfile', maxCount: 1},
+  {name: 'sponsorfile', maxCount: 1},
+  {name: 'eventfile', maxCount: 1},
+  {name: 'locationfile', maxCount: 1}
+]);
+
 app.set('port', (process.env.PORT || 5000));
 
 // Use the www folder as static frontend
 app.use('/', express.static(__dirname + '/www'));
 
-app.all('/generate', function(req, res) {
+app.post('/generate', uploadedFiles, function(req, res, next) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ['next' is defined but never used.](https://www.codacy.com/app/triplez/open-event-webapp/file/2729341163/issues/source?bid=3233857&fileBranchId=3349103#l25)